plugins/woocommerce/src/Internal/DataStores/Orders/OrdersTableDataStore.php

```diff
@@ -568,21 +644,22 @@ private function set_new_order_email_sent( $order, $set, $save = true ) {
 	 * @return bool Whether stock was reduced.
 	 */
 	public function get_stock_reduced( $order ) {
-		$order = is_numeric( $order ) ? wc_get_order( $order ) : $order;
-		return wc_string_to_bool( $order->get_meta( '_order_stock_reduced', true ) );
+		$order = is_int( $order ) ? wc_get_order( $order ) : $order;
```

Comment:
@vedanshujain This breaks backward compatibility. The implementation of this method for CPT accepts a numeric string. It may be better to use `WC_Order_Factory::get_order_id()` just like `WC_Order_Data_Store_CPT::get_stock_reduced()` does.